aif360/sklearn/datasets/seaphe_datasets.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can you add a line explaining where the code is copied from and which parts this covers?

Reply:
I used the code from :-
https://github.com/microsoft/tempeh/blob/main/tempeh/datasets/seaphe_datasets.py

This was mentioned on the issue assigned to me :-

https://github.com/Trusted-AI/AIF360/issues/359

Reply:
I meant write it in the source file :)

I just want it to be clear which parts are covered by that copyright and which are written by us

Reply:
Done. Mentioned it in "seaphe_datasets" before the "fetch_lawschool_gpa" method.